solr/core/src/java/org/apache/solr/search/SolrIndexSearcher.java

```diff
@@ -1806,7 +1813,7 @@ private void getDocListNC(QueryResult qr, QueryCommand cmd) throws IOException {
     int last = len;
```

Comment:
existing code: personally I would find more `final` use in this method helpful, #2258 proposes to add it, I'm happy to resolve any merge conflicts that this may cause for the pull request here.